dragonfly_doe2/doe/hvac.py

```diff
@@ -13,3 +16,11 @@ def to_inp(self):
 
     def __repr__(self):
         return self.to_inp()
+
+
+@dataclass
+class _HVACBase(object):
```

Comment:
You don't need `object` in Python 3.

```suggestion
class _HVACBase:
```